sql/information_schema/information_schema.go

```diff
@@ -30,6 +32,21 @@ import (
 	"github.com/dolthub/go-mysql-server/sql/plan"
 )
 
+func init() {
```

Comment:
Don't do this if you can avoid it. You should be able to set this directly in the engine at test setup time.

Reply:
Part of my session / transaction changes puts the information_schema db in the Catalog, so you could just set it there during test setup.